py_zipkin/logging_helper.py

```diff
@@ -89,6 +91,7 @@ def emit_spans(self):
             return
 
         span_sender = ZipkinBatchSender(self.transport_handler,
+                                        self._get_tracer(),
```

Comment:
Can you pass the getter function for consistency? It's probably fine to pass the tracer object itself here, but that'd break if we will ever make ZipkinBatchSender asynchronous.